config/config.js

```diff
@@ -15,22 +15,24 @@
  * Authors: 
  * 		Ghada Gharbi < ghada.gharbi@sensinov.com >
  ******************************************************************************/
-const serverPort = 3000;
-const serverHostHTTP = 'http://127.0.0.1:' + serverPort + '/';
+const serverPort = int(process.env.SERVER_PORT) || 3000;
```

Comment:
`const serverPort = +process.env.SERVER_PORT || 0 || 3000;`

Reply:
In this case the || 0 || doesn't make sense, so simply:
```suggestion
const serverPort = +process.env.SERVER_PORT || 3000;
```
should do. It covers all the NaN possibilities with default 3000.